packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -140,257 +143,257 @@ const registerCommands = (
     'sf.org.delete.default',
     orgDelete
   );
-  const orgDeleteUsernameCmd = vscode.commands.registerCommand(
+  const orgDeleteUsernameCmd = registerCommand(
     'sf.org.delete.username',
     orgDelete,
     { flag: '--target-org' }
   );
-  const orgDisplayDefaultCmd = vscode.commands.registerCommand(
+  const orgDisplayDefaultCmd = registerCommand(
     'sf.org.display.default',
     orgDisplay
   );
-  const orgDisplayUsernameCmd = vscode.commands.registerCommand(
+  const orgDisplayUsernameCmd = registerCommand(
     'sf.org.display.username',
     orgDisplay,
     { flag: '--target-org' }
   );
-  const orgListCleanCmd = vscode.commands.registerCommand(
+  const orgListCleanCmd = registerCommand(
     'sf.org.list.clean',
     orgList
   );
-  const dataQueryInputCmd = vscode.commands.registerCommand(
+  const dataQueryInputCmd = registerCommand(
     'sf.data.query.input',
     dataQuery
   );
-  const dataQuerySelectionCmd = vscode.commands.registerCommand(
+  const dataQuerySelectionCmd = registerCommand(
     'sf.data.query.selection',
     dataQuery
   );
-  const projectGenerateCmd = vscode.commands.registerCommand(
+  const projectGenerateCmd = registerCommand(
     'sf.project.generate',
     sfProjectGenerate
   );
 
-  const packageInstallCmd = vscode.commands.registerCommand(
+  const packageInstallCmd = registerCommand(
     'sf.package.install',
     packageInstall
   );
-  const projectGenerateWithManifestCmd = vscode.commands.registerCommand(
+  const projectGenerateWithManifestCmd = registerCommand(
     'sf.project.generate.with.manifest',
     projectGenerateWithManifest
   );
 
-  const apexGenerateTriggerCmd = vscode.commands.registerCommand(
+  const apexGenerateTriggerCmd = registerCommand(
     'sf.apex.generate.trigger',
     apexGenerateTrigger
   );
 
-  const startApexDebugLoggingCmd = vscode.commands.registerCommand(
+  const startApexDebugLoggingCmd = registerCommand(
     'sf.start.apex.debug.logging',
     startApexDebugLogging
   );
 
-  const stopApexDebugLoggingCmd = vscode.commands.registerCommand(
+  const stopApexDebugLoggingCmd = registerCommand(
     'sf.stop.apex.debug.logging',
     stopApexDebugLogging
   );
 
-  const isvDebugBootstrapCmd = vscode.commands.registerCommand(
+  const isvDebugBootstrapCmd = registerCommand(
     'sf.debug.isv.bootstrap',
     isvDebugBootstrap
   );
 
-  const configSetCmd = vscode.commands.registerCommand(
+  const configSetCmd = registerCommand(
     'sf.config.set',
     configSet
   );
 
-  const diffFile = vscode.commands.registerCommand('sf.diff', sourceDiff);
+  const diffFile = registerCommand('sf.diff', sourceDiff);
 
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const diffFolder = vscode.commands.registerCommand(
+  const diffFolder = registerCommand(
     'sf.folder.diff',
     sourceFolderDiff
   );
 
-  const forceRefreshSObjectsCmd = vscode.commands.registerCommand(
+  const forceRefreshSObjectsCmd = registerCommand(
```

Comment:
How would you feel about only doing this for sobject refresh for the initial release, but expanding to other services after we've had a couple of week to ensure there are no surprises with just wrapping sobjectRefresh?

Reply:
This is a great idea @gbockus-sf

Reply:
👍